import-export-cli/integration/testutils/throttlePolicy_testUtils.go

```diff
@@ -0,0 +1,173 @@
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[PolicyNameKey])
+
+	exportedOutput, _ := exportThrottlePolicy(t, policyName, args.SrcAPIM.GetEnvName())
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+	args.SrcAPIM.DeleteThrottlePolicy(fmt.Sprintf("%v", args.Policy[PolicyIDKey]), policyType)
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	_, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, policyName, policyType)
+	// Validate env 1 and env 2 policy is equal
+	ValidatePoliciesEqual(t, args, importedPolicy)
+	RemoveExportedThrottlingPolicyFile(t, args.ImportFilePath)
+}
+
+// Adds a new Throttling Policy to an env
+func AddNewThrottlePolicy(t *testing.T, client *apim.Client, username, password, policyType string) interface{} {
+	client.Login(username, password)
+	generatedPolicy := client.GenerateSampleThrottlePolicyData(policyType)
+	addedPolicy := client.AddThrottlePolicy(t, generatedPolicy, policyType)
+	return addedPolicy
+}
+
+// Exports Throttling Policy from an env
+func exportThrottlePolicy(t *testing.T, name, env string) (string, error) {
+	output, err := base.Execute(t, "export", "policy", "rate-limiting", "-n", name, "-e", env, "-k", "--verbose")
+	return output, err
+}
+
+// Imports Throttling policy to an env
+func importThrottlePolicy(t *testing.T, args *ThrottlePolicyImportExportTestArgs) (string, error) {
+	var output string
+	var err error
+	if args.Update {
+		output, err = base.Execute(t, "import", "policy", "rate-limiting", "-e", args.DestAPIM.GetEnvName(), "-f", args.ImportFilePath, "-u")
+	} else {
+		output, err = base.Execute(t, "import", "policy", "rate-limiting", "-e", args.DestAPIM.GetEnvName(), "-f", args.ImportFilePath)
+	}
+	return output, err
+}
+
+// Retrieve Throttling policy by name
+func getThrottlingPolicyByName(t *testing.T, args *ThrottlePolicyImportExportTestArgs, throttlePolicyName, policyType string) (map[string]interface{}, error) {
+	client := args.DestAPIM
+	uuid := client.GetThrottlePolicyID(t, args.Admin.Username, args.Admin.Password, throttlePolicyName, policyType)
+	policy := client.GetThrottlePolicy(uuid, policyType)
+	client.DeleteThrottlePolicy(uuid, policyType)
+	return ThrottlePolicyStructToMap(policy)
+}
+
+// Validates whether throttling policies are equal
+func ValidatePoliciesEqual(t *testing.T, args *ThrottlePolicyImportExportTestArgs, importedPolicy map[string]interface{}) {
```

Comment:
Can't we make this function private by starting with a lowercase letter? Seems this function only has usages within the same package.

Reply:
fixed with 73381a57ed995c647ea2aa52394a7982f008363c